autobotAI_integrations/integrations/nessus/__init__.py

```
@@ -152,18 +152,44 @@ def get_details(cls):
+            base_url=url,
+            headers={
+                "X-ApiKeys": f"accessKey={access_key}; secretKey={secret_key}",
+                "Accept": "application/json",
+                "Content-Type": "application/json",
+            },
+            verify_ssl=verify_ssl,
+        )
+
     @staticmethod
     def get_python_sdk_client(
         connection_details: Union[RestAPICreds, SDKCreds],
```

Ruff (0.15.20)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)